api/v1alpha1/databasecluster_types.go

```diff
@@ -187,22 +186,6 @@ type PMMSpec struct {
 	Password      string `json:"password,omitempty"`
 }
 
-// Monitoring contains monitoring settings.
-type Monitoring struct {
-	// Enabled is a flag to enable monitoring
-	Enabled   bool                        `json:"enabled"`
-	PMM       *PMMSpec                    `json:"pmm,omitempty"`
-	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
Was the removal of the resources field a conscious decision? Wouldn't it be useful to be able to set resource limits for the monitoring container?

Reply:
No, this is a good catch.
I did not see it in the CR samples and thought it's not available, but it is. Added back.